src/psyclone/tests/psyGen_test.py

```diff
@@ -546,23 +546,60 @@ def test_derived_type_deref_naming():
 end module dummy_mod
 '''
 
-# Schedule class tests
 
+# Schedule class tests
 
 def test_sched_view(capsys):
-    ''' Check the view method of the Schedule class. We need a Schedule
-    object for this so go via the dynamo0.3 sub-class '''
+    ''' Check the view method of the Schedule class'''
     from psyclone import dynamo0p3
-    from psyclone.psyGen import colored, SCHEDULE_COLOUR_MAP
+    from psyclone.psyGen import Schedule, colored, SCHEDULE_COLOUR_MAP
     _, invoke_info = parse(os.path.join(BASE_PATH,
                                         "15.9.1_X_innerproduct_Y_builtin.f90"),
                            api="dynamo0.3")
     psy = PSyFactory("dynamo0.3", distributed_memory=True).create(invoke_info)
-    super(dynamo0p3.DynSchedule, psy.invokes.invoke_list[0].schedule).view()
+
+    # For this test use the generic class
+    psy.invokes.invoke_list[0].schedule.__class__ = Schedule
+    psy.invokes.invoke_list[0].schedule.view()
+
     output, _ = capsys.readouterr()
     assert colored("Schedule", SCHEDULE_COLOUR_MAP["Schedule"]) in output
 
 
+def test_sched_can_be_printed():
+    ''' Check the schedule class can always be printed'''
+    from psyclone import dynamo0p3
```

Comment:
pylint complains that dynamo0p3, colored and SCHEDULE_COLOUR_MAP are unused.

Reply:
removed them